Automattic/wp-calypso
# CRO i5: Add need more info box with scrollToElement.

### Changes proposed in this Pull Request

This PR adds a new "Need more info?" box below the plan bundles.  The "Compare all product bundles" button scrolls the viewport down to the Bundle Comparison section.

Implementation note.
- Disregard the red button colors in Calypso. The Calypso colors are changed to Jetpack green in a previous PR that will be merging soon.

### Testing instructions

- Download the PR
- Run both Calypso and Jetpack cloud concurrently
- Select the `i5 - Saas table design` variant of the a/b test jetpackConversionRateOptimization
- Visit the Plans page
- In both Calypso and Jetpack Cloud:
    - Verify that the "Need more info" box (See screenshot) looks like the Figma design.
    - Click the "Compare all product bundles" button and verify that the viewport scrolls down to the "Bundle Comparison" section.

### Screenshot

![Markup 2020-11-11 at 12 50 56](https://user-images.githubusercontent.com/11078128/98863809-c4d6e000-241d-11eb-891e-958dd1bceb65.png)


Fixes 1196341175636977-as-1199149328812604
